pkg/controller/nmstate/nmstate_controller.go

```diff
@@ -5,11 +5,14 @@ import (
 	"os"
 	"path/filepath"
 
+	yaml "github.com/nmstate/kubernetes-nmstate/pkg/helper/render"
```

Comment:
Don't rename it, use it as `render.ToYaml`

Reply:
There is `render` package already